cmd/operator/main.go

```diff
@@ -86,6 +88,17 @@ func init() {
 	utilruntime.Must(certtypes.AddToScheme(scheme))
 }
 
+func doesManagedClusterCRDExist(mgr manager.Manager) error {
```

Comment:
Lets distinguish between an actual error fetching the CRD to a `false` answer on whether it exists

Reply:
If you look at the implementation, you can see that in the error cases, this is where we would return 'false'.
So any error is the same as saying 'false', no need to change this.

```
func doesManagedClusterCRDExist(mgr manager.Manager) (bool, error) {
	gvk, err := apiutil.GVKForObject(&clusterv1.ManagedCluster{}, mgr.GetScheme())
	if err != nil {
		return false, err
	}
	if _, err = mgr.GetRESTMapper().RESTMapping(gvk.GroupKind(), gvk.Version); err != nil {
		return false, err
	}
	return true, nil
}
```